recipes/teneto/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "teneto" %}
+{% set version = "0.5.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wiheto/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: b74daefe65c34f200b292eb3cd21d67f357fbd892c7b678c35e772af2b1eb1a2
+
+build:
+  script: python -m pip install . -vv
+  number: 0
+  skip: True  # [win]
```

Comment:
Why not noarch?

Reply:
Package is doing some file opening that has a unicode issue

Reply:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1100738&view=logs&j=53c3d5f4-e7c7-5b53-4b94-211701b2ba17&t=8f042650-8479-5761-7a4a-8d021c0bb2de

Reply:
That is (usually) fine. The final package may still be functional on Windows. If not, the alternative is to make a OS-noarch. See https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-packages-with-os-specific-dependencies

I believe this one is OK with the first option.

Reply:
Works for me!  It's noarch again